javascript/builtins/Array.json

```diff
@@ -2256,7 +2256,14 @@
                 "version_added": false
               },
               "nodejs": {
-                "version_added": false
+                "version_added": "6.5.0",
+                "notes": "<code>--harmony-array-prototype-values</code> is in progress and is not enabled by the <code>--harmony</code> flag",
```

Comment:
Tiny, tiny thing: would be nice to have a period/full stop at the end of the sentence here. After that, we'll be good to merge. Thank you!

Reply:
Sure, no problem. Done. :-)